_posts/en/newsletters/2020-03-11-newsletter.md

```diff
@@ -0,0 +1,146 @@
+---
+title: 'Bitcoin Optech Newsletter #88'
+permalink: /en/newsletters/2020/03/11/
+name: 2020-03-11-newsletter
+slug: 2020-03-11-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter links to a description of methods for preventing
+hardware wallets from leaking private information through transaction
+signatures and provides an update on the BIP322 generic signmessage
+protocol.  Also included are our regular sections about new releases and
+notable merges of popular Bitcoin infrastructure projects.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Exfiltration resistant nonce protocols:** Pieter Wuille sent an
```

Comment:
Exfiltration-resistant

Reply:
I don't think the hyphen is necessary, and aesthetically I prefer not having it. See https://github.com/bitcoinops/bitcoinops.github.io/pull/359#discussion_r386676690 where we discussed what nomenclature to use and hyphenation.

Reply:
I believe this case is different because "exfiltration" modifies only "resistant" to indicate the type of resistance, whereas in the other case I think "anti" modified the phrase. But I won't bikeshed further :smile: